apps/omg_watcher/test/fixtures.exs

```diff
@@ -274,4 +269,24 @@ defmodule OMG.Watcher.Fixtures do
     |> Enum.with_index()
     |> Enum.map(fn {recovered_tx, txindex} -> {blknum, txindex, recovered_tx.tx_hash, recovered_tx} end)
   end
+
+  defp ensure_web_started(module, function, args, 0), do: apply(module, function, args)
+
+  defp ensure_web_started(module, function, args, counter) do
+    {:ok, pid} = apply(module, function, args)
+    {:ok, pid}
```

Comment:
just for my understanding, but we want to pattern match to ensure this does properly starts. But why pass the same result tuple down the chain again? Why not just the `pid` at this point? 

We also [pattern match](https://github.com/omisego/elixir-omg/pull/1093/files#diff-43636fce1bc9dcf250c274eddb21c7b9R150) for this result elsewhere so wondering if we should just simply pass it through and:

```elixir
defp ensure_web_started(module, function, args, counter), do: apply(module, function, args)
```

Reply:
it's because I don't want to act like I do anything with the output of the `apply/3`. I want to return the raw result.